analyses/molecular-subtyping-pathology/pathology_free_text-subtyping-jxg.nb.html

```diff
@@ -3169,24 +3163,22 @@ <h2>Session Info</h2>
 [1] stats     graphics  grDevices utils     datasets  methods   base     
 
 other attached packages:
- [1] forcats_0.4.0   stringr_1.4.0   dplyr_1.0.7     purrr_0.3.2    
+ [1] forcats_0.4.0   stringr_1.4.0   dplyr_0.8.3     purrr_0.3.2    
  [5] readr_1.3.1     tidyr_0.8.3     tibble_2.1.3    ggplot2_3.2.0  
  [9] tidyverse_1.2.1 magrittr_1.5   
 
 loaded via a namespace (and not attached):
- [1] tidyselect_1.1.1 xfun_0.8         haven_2.1.1      lattice_0.20-38 
- [5] colorspace_1.4-1 vctrs_0.3.8      generics_0.0.2   htmltools_0.3.6 
- [9] yaml_2.2.0       base64enc_0.1-3  utf8_1.1.4       rlang_0.4.12    
-[13] pillar_1.6.4     glue_1.5.0       withr_2.1.2      DBI_1.0.0       
-[17] modelr_0.1.4     readxl_1.3.1     lifecycle_1.0.1  munsell_0.5.0   
-[21] gtable_0.3.0     cellranger_1.1.0 rvest_0.3.4      evaluate_0.14   
-[25] knitr_1.23       fansi_0.4.0      broom_0.5.2      Rcpp_1.0.1      
-[29] scales_1.0.0     backports_1.1.4  jsonlite_1.6     hms_0.4.2       
-[33] digest_0.6.20    stringi_1.4.3    grid_3.6.0       cli_1.1.0       
-[37] tools_3.6.0      lazyeval_0.2.2   crayon_1.3.4     pkgconfig_2.0.2 
-[41] ellipsis_0.3.2   xml2_1.2.0       lubridate_1.7.4  assertthat_0.2.1
-[45] rmarkdown_1.13   httr_1.4.0       rstudioapi_0.10  R6_2.4.0        
-[49] nlme_3.1-140     compiler_3.6.0  </code></pre>
+ [1] Rcpp_1.0.1       cellranger_1.1.0 pillar_1.4.2     compiler_3.6.0  
+ [5] base64enc_0.1-3  tools_3.6.0      digest_0.6.20    lubridate_1.7.4 
+ [9] jsonlite_1.6     evaluate_0.14    nlme_3.1-140     gtable_0.3.0    
+[13] lattice_0.20-38  pkgconfig_2.0.2  rlang_0.4.0      cli_1.1.0       
+[17] rstudioapi_0.10  yaml_2.2.0       haven_2.1.1      xfun_0.8        
+[21] withr_2.1.2      xml2_1.2.0       httr_1.4.0       knitr_1.23      
+[25] generics_0.0.2   hms_0.4.2        grid_3.6.0       tidyselect_0.2.5
+[29] glue_1.3.1       R6_2.4.0         readxl_1.3.1     rmarkdown_1.13  
+[33] modelr_0.1.4     backports_1.1.4  scales_1.0.0     htmltools_0.3.6 
+[37] rvest_0.3.4      assertthat_0.2.1 colorspace_1.4-1 stringi_1.4.3   
+[41] lazyeval_0.2.2   munsell_0.5.0    broom_0.5.2      crayon_1.3.4    </code></pre>
```

Comment:
are you rerunning this _inside_ the docker container? I would not expect all of these package changes, unless this is due to the move off of MRAN and this wasn't run since before then...